lib/install/management/appliance.go

```diff
@@ -146,7 +149,7 @@ func (d *Dispatcher) getName(vm *vm.VirtualMachine) string {
 	return name
 }
 
-func (d *Dispatcher) deleteVM(vm *vm.VirtualMachine, force bool) error {
+func (d *Dispatcher) deleteVM(vm *vm.VirtualMachine, conf *config.VirtualContainerHostConfigSpec, force bool) error {
```

Comment:
this method is called for both appliance vm delete and container vm delete, I think we should not start from here to detect if the folder is same to volume store